[#63] 닉네임 중복 체크

namewhat99 opened the pull request:
## 이슈
- #63

namewhat99 committed:
Fix: 닉네임 중복 체크

koomin1227 committed:
Merge branch 'develop' into feature/nicknameDup-#63

koomin1227 reviewed (changes requested):
nickname 중복을 불허 할 거면 entity 에서 nickname column에 유니크 설정하는 것이 좋을 것 같습니다...

namewhat99 committed:
Fix: 닉네임 column unique 설정

namewhat99 committed:
Feat : 분기된 브랜치 pull
Merge branch 'feature/nicknameDup-#63' of https://github.com/Boost-Coder/Backend into feature/nicknameDup-#63
